src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -9,6 +9,10 @@ import {
   START_NEW_APP_DEFAULT_MESSAGE,
   APP_TYPE_DEFAULT,
 } from '../../forms-system/src/js/constants';
+import {
+  WIZARD_STATUS,
+  WIZARD_STATUS_RESTARTING,
+} from 'applications/static-pages/wizard';
```

Comment:
Having the platform code import this module has the downstream effect of importing the `applications/disability-benefits/all-claims/config/selectors` module as well, which then makes platform code—and all applications using this—dependent on code from a single application.

We need to, as much is possible, not have platform code depend on application code.

My recommendation: Pull these constants out into their own module and import them from here and `applications/static-pages/wizard/index.js`.

Reply:
Added a new folder + files in `platform/site-wide/wizard`